voltohandson/quickstart.rst

```diff
@@ -36,46 +44,51 @@ To build your Volto site, use the ``make`` command:
 
     .. code-block:: console
 
-      $ make start-backend-docker
+        $ make start-backend-docker
+
+
+Yarn environment
+----------------
 
 The repo also has a Volto project ready to build in the root.
-You should have all the :ref:`Volto dependencies <install-deps-volto-label>` installed on your machine.
-To build it you should do:
+
+First install :ref:`Volto dependencies <install-deps-volto-label>` on your machine.
+
+Then to build the yarn environment, you can either use:
 
 .. code-block:: console
 
-  $ make build-frontend
+    $ make build-frontend
 
-or just:
+or:
 
 .. code-block:: console
 
     $ yarn
 
+
 Executing environments
 ----------------------
 
-Use this command to run Plone:
+In your Plone environment, use this command to run Plone:
 
 .. code-block:: console
 
-  $ make start-backend
+    $ make start-backend
 
-and this command to run Volto:
+In your yarn environment, use this command to run Volto:
```

Comment:
```suggestion
In your Yarn environment, use this command to run Volto:
```
Yarn is a name, right?

Reply:
I always have to check their website or package to verify proper case. You got these right.